explain.py

```diff
@@ -155,6 +155,14 @@ def _explain_hash(self) -> Tuple[str, Dict[str, str]]:
                             "Buckets are doubled until there are enough, so they are always a power of 2."
         }, **self._generic_explain_dict())
 
+    def _explain_sorted_merge(self) -> Tuple[str, Dict[str, str]]:
+       return f"A sorted merge join is performed on {self.hash_cond}.", dict({
+            "Description": "Sorted Merge Join is when two lists are sorted on their join keys before being joined together. "
+                           "Postgres then traverse over the two lists in order, finding pairs that have identical join keys"
+                           " and returning them as a new, joined row.\n",
+            "Join type": self.join_type
+        }, **self._generic_explain_dict()) 
+
```

Comment:
Also, update explainMapping, so sort merge node type is recognised, you must get the exact string value of sort merge node type.